app/src/main/java/com/sun/wsm_android_v2/data/request/LoginRequest.kt

```diff
@@ -0,0 +1,3 @@
+package com.sun.wsm_android_v2.data.request
+
+class LoginRequest(var email: String, var password: String)
```

Comment:
chung luôn 1 tên đi e
```suggestion
class SignInRequest(var email: String, var password: String)
```